### DISCUSSION TITLE
Python Editor Column 80 Visual Aid

### DISCUSSION THREAD

**Discussion by vsfeedback** [Microsoft/PTVS#7956] [Enchancements]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/t/Python-Editor-Column-80-Visual-Aid/10691729)._

---
Since Python coding best practices (PEP 8) recommends keeping line length to less than 80 characters, draw a faint medium gray vertical line between columns 79 and 80 when editing Python code. Obviously, that could be a configurable option.

---
#### Feedback Bot on 6/27/2024, 00:16 AM: 

(private comment, text removed)

**Comment by StellaHuang95** [comment]
This would be similar to the `editor.ruler` setting in vscode that renders vertical rulers after a certain number of characters. As this is more of an editor feature rather than a PTVS-specific feature, we will transfer this proposal to discussion for now. If it receives more upvotes, we can bring it up to the editor team for consideration.
